cfme/rest/__init__.py

```diff
@@ -354,3 +355,23 @@ def mark_vm_as_template(rest_api, provider, vm_name):
         lambda: rest_api.collections.templates.find_by(name=vm_name).subcount != 0,
         num_sec=700, delay=15)
     return rest_api.collections.templates.get(name=vm_name)
+
+
+def arbitration_settings(request, rest_api, num=2):
```

Comment:
I don't like the fact that the rest module is creating the random data. The test should create the random data and pass the ```body``` to the ```arbitration_settings()``` function which then just creates it with whatever the body is.